strings/cas_white.txt

```diff
@@ -151,27 +151,27 @@ Immulsions.
 100 potency bluespace bananas.
 The crotchety IBM piece of shit you're supposed to call an AI.
 An irritatingly chipper borg.
-The lizard fuckpile.
+A lizard clique that refuses to speak to non-lizards.
```

Comment:
```suggestion
A vuulek clique that refuses to speak to non-vuulen.
```

Reply:
the fuck is a vuleek

Reply:
lore writers, not even once.

Reply:
considering this is is a  funny card game, why refer to them with a formal name no one remembers?

Reply:
why does it matter that they not get called by their formal names? They very much are in other areas, i.e. traitor objectives, so why not make it more consistent.

Reply:
i dont agree but i'll change it because i dont care enough to argue